website/pages/how-to-guides/update-plugins-using-renovate.md

```diff
@@ -0,0 +1,139 @@
+
+While Renovate provides tons of built-in functionality to make most dependency management simple, it also provides a powerful functionality called 'RegexManager' which allows you to leverage custom regular expression statements to manually extract dependencies, their version, and updating mechanism. Click [here](https://docs.renovatebot.com/modules/manager/regex/) for more details on the specific configurations of RegexManager.
+{/*<!-- vale off -->*/}
+For our purposes, we're concerned about the following items:
+- **datasourceTemplate**: datasource is where the package's versions are managed.
+- **packageNameTemplate**: packageName for the datasource to use.
+- **depNameTemplate**: depName being the 'friendly' name that is shown in renovate PRs.
+- **extractVersionTemplate**: regular expression for discovering plugin version in the datasource.
+- **versioningTemplate**: what versioning 'format' the datasource follows
+- **fileMatch**: path to the cloudquery.yaml file in your repository.
+- **matchStrings**: regular expression to find the plugins in your cloudquery.yaml file.
+{/*<!-- vale on -->*/}
+Using a simplistic configuration file that looks like:
+
+```yaml copy
+---
+kind: source
+spec:
+  name: gcp
+  path: cloudquery/gcp
+  version: "v5.4.0"
+  tables: ["*"]
+  sestinations: ["postgresql"]
+---
+kind: source
+spec:
+  name: aws
+  path: cloudquery/aws
+  version: "v9.1.0"
+  destinations: ["postgresql"]
+  tables: ["*"]
+---
+kind: destination
+spec:
+  name: postgresql
+  path: cloudquery/postgresql
+  version: "v2.0.1"
+  write_mode: overwrite-delete-stale
+``` 
+
+You can see that we have three 'keys', meaning fields that are static across all plugins, we can leverage these to construct our custom regex statement for each plugin. These 'keys' will be used as regular expression named capture group. Those fields being:
+
+- **kind**: what type of plugin this is (source or destination).
+- **path**: the 'name' of the plugin (such as gcp or aws).
+- **version**: the current version of the plugin.
+
+### Step 2: **Create Custom RegexManager**
+
+Now that we've identified the fields we need to extract from the configuration file to create the custom regex, next we need to construct the regex statement to extract those 'keys' into named capture groups.
+
+For this it's recommended to use a regex testing tool to ensure your regex is actually working. [Regex101](https://regex101.com/) is a popular one but feel free to use any others as long as they support the PCRE2 regex engine. So taking our sample configuration file from above, we begin to play around with various regex until we meet the following criteria:
+
+- a named capture group for the plugin type (source or destination) which we will reference as 'kind' in this tutorial.
+- a named capture group for the plugin name (aws, gcp, etc..) which we will reference as 'plugin' in this tutorial.
+- a named capture group for the plugin current version which **must** be named currentValue.
```

Comment:
```suggestion
- a named capture group for the plugin current version which **must** be named `currentValue`.
```

Putting this inside backticks should keep the linter happy

Reply:
sorry - just saw this after my last commit, I added them to the `.github/styles/proselint/Vocab/Base/accept.txt` file. Want me to back that commit out and backtick them instead?